bin/qless-js-worker.js

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env node
+
+const assert = require('assert');
+const qless = require('qless-js');
+
+var commander = require('commander');
+
+function collect(name, queues) {
+  queues.push(name);
+  return queues;
+}
+
+commander
+  .option('-h, --host <url>', 'The redis:// url to connect to')
+  .option('-n, --name <name>', 'The name to identify your worker as')
+  .option('-d, --workdir <dir>', 'The base work directory path')
+  .option('-c, --concurrency <n>', 'The number of concurrent jobs to run [default 10]')
+  .option('-i, --interval <n>', 'Polling interval in seconds [default 60]', parseInt)
+  .option('-q, --queue <name>', 'Add a queue to work on', collect, [])
```

Comment:
@evanbattaglia -- My linter doesn't mind the missing semicolon on this line. Do y'all have a `.eslintrc` config you'd like me to use? I'm happy to adhere to whatever style, so long as it it's lint-configured.